### ISSUE TITLE
Resize examples screenshots

### ISSUE THREAD

**Issue by XhmikosR** [twbs/bootstrap#29030]
It seems after some recent changes, the examples screenshots dimensions have changed.

We shouldn't have bigger images than what we need them to be.

**Comment by mdo** [comment]
Agreed, but let's wait on this until we're much closer to release. I'm not 100% on the design and layout just yet.

**Comment by kodeGenius** [comment]
Hi all, anyone working on this?  I'll be glad to take it on.

**Comment by mdo** [comment]
Closing as a duplicate of #30286 just to consolidate.
